datamodelutils/postgres_admin.py

```diff
@@ -28,66 +28,22 @@
 )
 
 logging.basicConfig()
-logger = logging.getLogger("gdc_postgres_admin")
+logger = logging.getLogger("postgres_admin")
 logger.setLevel(logging.INFO)
 
 name_root = "table_creator_"
 app_name = "{}{}".format(name_root, random.randint(1000, 9999))
```

Comment:
The removal of the killing-blocker code replicates [this PR](https://github.com/uc-cdis/gen3datamodel/pull/43) for gen3datamodel and [this PR](https://github.com/NCI-GDC/gdcdatamodel/pull/331) for gdcdatamodel.